README.md

```diff
@@ -1,59 +1,43 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Todo List web app
 
-> One paragraph statement about the project.
+> Todo list app where you can add your tasks.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![screenshot](./screenshot.png)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JAVASCRIPT
+- WEBPACK
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://youmari.github.io/todo-list)
```

Comment:
[optional]

- The live link is not working. Please fix it or remove the section from the README.